### PR TITLE
stable/rabbitmq-ha Bump rabbit version

### PR DESCRIPTION
**Pull request by wjdavis5**
#### What this PR does / why we need it:
Bump rabbit version to latest bug fix

#### Special notes for your reviewer:
@steven-sheehy 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
-appVersion: 3.7.15
-version: 1.33.0
+appVersion: 3.7.19
+version: 1.34.0
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: `1.33.1` since just bugfix bump

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Also update version in README

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (wjdavis5) [reply stable/rabbitmq-ha/Chart.yaml]: Done.

**Review by steven-sheehy — APPROVED** [review]
/lgtm

**Pull request merged**